tests/test_callbackcontext.py

```diff
@@ -105,3 +105,13 @@ def test_match(self, cdp):
         callback_context.matches = ['test', 'blah']
 
         assert callback_context.match == 'test'
+
+    def test_data_assignment(self, cdp):
+        update = Update(0, message=Message(0, User(1, 'user', False), None, Chat(1, 'chat')))
+
+        callback_context = CallbackContext.from_update(update, cdp)
+
+        with pytest.raises(AttributeError):
```

Comment:
Please use one close per context manager. (so three `with pytest.raises`) here.
Now even if one of them raises and the rest don't the test passes.

Reply:
oh really? Okay, sad. Fixed